remake.yml

```diff
@@ -7,20 +7,26 @@ packages:
   - rnaturalearth
   - cowplot
   - scipiper
+  - leaflet
+  - leafpop
+  - htmlwidgets
 
 sources:
   - 1_fetch/src/find_oldest_sites.R
   - 1_fetch/src/get_site_data.R
   - 123_state_tasks.R
   - 3_visualize/src/map_sites.R
+  - 3_visualize/src/plot_data_coverage.R
+  - 3_visualize/src/map_timeseries.R
 
 
 targets:
   main:
     depends:
-      - state_tasks
+      - obs_tallies
```

Comment:
you can take out obs_tallies now that data_coverage.png is in this list (because data_coverage.png requires obs_tallies and is the ultimate reason that obs_tallies needs to be built)